### PR TITLE
Cleaning up after WebView implementation

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Since we are moving towards the [WebView implementation of wallet features](https://github.com/litentry/litentry-app/pull/1236) we have some clean-up opportunities in the code.

For example removing the `react-native-substrate-sign` library and all of its implementations is one of them. Additionally since we are managing application state using [recoil](https://recoiljs.org/) now, we can get rid of the Accounts context provider too. And since because of an implementation detail of moving the selected network state into a persisted recoil atom we can actually get rid of the Network context provider as well.

This PR mainly takes care of these clean-ups.

- [x] Cleaning up react-native-substrate-sign
- [x] Removing Accounts context provider
- [x] Moving active network state into recoil atom
- [x] Removing Network context provider
- [x] Changing firebase remote config initialisation process

### CODE REVIEW

**Comment by geastwood** [comment]
This is good effort, however can we also have a clear task list to provide better overview on what we trying to achieve within this PR?

File: src/polkadotApi/useAppAccounts.ts [hunk 1]
```diff
@@ -1,16 +1,19 @@
-import {useMemo} from 'react';
-import {useNetwork} from 'context/NetworkContext';
-import {useRecoilState} from 'recoil';
+import {networkState} from '@atoms/network';
+import {selector, useRecoilState, useRecoilValue} from 'recoil';
 import {appAccountsState} from './atoms';
 
+const netWorkAccountState = selector({
```
Comment (silva-fj) [inline src/polkadotApi/useAppAccounts.ts]:
```suggestion
const networkAccountsState = selector({
```

File: src/ui/screens/Council/MotionsScreen.tsx [hunk 2]
```diff
@@ -16,14 +16,14 @@ import {SelectAccount} from '@ui/components/SelectAccount';
...
-import type {Account as LocalAccount} from 'context/AccountsContext';
-import {useNetwork} from 'context/NetworkContext';
+import {useNetwork} from '@atoms/network';
 import type {SupportedNetworkType} from 'src/types';
 import {Caption, Card, Divider} from 'react-native-paper';
 import {ProposalCall} from '@ui/components/ProposalCall';
 import {ItemRowBlock} from '@ui/components/ItemRowBlock';
 import {AccountTeaser} from '@ui/components/Account/AccountTeaser';
 import {getProposalTitle} from 'src/utils/proposal';
+import type {Account as LocalAccount} from '@polkadotApi/types';
```
Comment (silva-fj) [inline src/ui/screens/Council/MotionsScreen.tsx]: We should probably make the types a bit more consistent, we use aliases for `Account`, `AppAccount` and `LocalAccount`. But we can tackle this in a separate issue

File: src/ui/screens/DevScreen.tsx [hunk 3]
```diff
@@ -5,11 +5,11 @@ import {InAppNotificationContent, InAppNotificationContext} from 'context/InAppN
 import {useApi} from 'context/ChainApiContext';
 import {ScrollView} from 'react-native-gesture-handler';
 import globalStyles, {standardPadding} from '@ui/styles';
-import {useNetwork} from 'context/NetworkContext';
+import {useNetwork} from '@atoms/network';
 import SafeView, {noTopEdges} from '@ui/components/SafeView';
 import {useConvictions} from 'src/api/hooks/useConvictions';
 import {SelectAccount} from '@ui/components/SelectAccount';
-import {useAccounts} from 'context/AccountsContext';
+import {useActiveAccount} from 'src/atoms/activeAccount';
```
Comment (silva-fj) [inline src/ui/screens/DevScreen.tsx]:
```suggestion
import {useActiveAccount} from '@atoms/activeAccount';
```

**Review by silva-fj — APPROVED** [review]

**Pull request merged**